web/src/navbar_alerts.ts

```diff
@@ -133,17 +140,16 @@ export function show_profile_incomplete(is_profile_incomplete) {
     }
 }
 
-export function get_demo_organization_deadline_days_remaining() {
-    const now = new Date(Date.now());
-    const deadline = new Date(realm.demo_organization_scheduled_deletion_date * 1000);
+export function get_demo_organization_deadline_days_remaining(): number {
+    const now = Date.now();
+    const deadline = new Date(realm.demo_organization_scheduled_deletion_date! * 1000).getTime();
```

Comment:
Should we remove this code and just use `realm.demo_organization_scheduled_deletion_date! * 1000` ?

Reply:
I'm not sure I understand the question? Or the choice to remove the `new Date()` wrapper?

Reply:
Yes, that's what I meant. Should we do that, or is this just fine?

Reply:
Subtracting UNIX timestamps seems fine.

Reply:
What justifies the unchecked `!`? There’s nothing that would prevent someone from changing code elsewhere to call this function while `realm.demo_organization_scheduled_deletion_date` is undefined, so this is not locally safe. Use `assert` instead.

Reply:
Done.